### PR TITLE
Added crypto logo to price module

### PR DESCRIPTION
**Pull request by AndreiMitrea**
Added logo to the crypto price module.

image

### CODE REVIEW

File: lib/tokens/tokenData.ts [hunk 1]
```diff
@@ -14,6 +14,8 @@ export interface ITokenMetadata {
   logo?: string
 }
 
+export type TokenInfo = Pick<PaymentMethod, 'tokenName' | 'tokenSymbol' | 'tokenLogo'> & {isContract: boolean};
+
```
Comment (motechFR) [inline lib/tokens/tokenData.ts:18]: Good cleanup!

File: components/common/CharmEditor/components/CryptoPrice.tsx [hunk 2]
```diff
@@ -162,7 +163,7 @@ export function CryptoPrice ({ preset, onQuoteCurrencyChange, onBaseCurrencyChan
               active={selectionList === 'base'}
               onClick={() => toggleSelectionList('base')}
             >
-              {getTokenInfo(paymentMethods, baseCurrency)?.tokenSymbol}
+              <CoinLogoAndTicker {...getTokenInfo(paymentMethods, baseCurrency)} />
```
Comment (motechFR) [inline components/common/CharmEditor/components/CryptoPrice.tsx:166]: Makes sense, good one adding this here

**Review by motechFR — APPROVED** [review]
Clean code all the way

File: components/common/CharmEditor/components/CryptoPrice.tsx (hunk 2 above)
Comment (AndreiMitrea) [reply components/common/CharmEditor/components/CryptoPrice.tsx:166]: Thank you!

**Pull request merged**